app/services/imo_api_service.rb

```diff
@@ -28,22 +28,29 @@ def send_invitation(patient, phone_number: nil)
     phone = phone_number || patient.latest_mobile_number
     locale = patient.locale
     url = IMO_BASE_URL + "send_invite"
+
+    unless locale.in?(["bn-BD", "en"])
+      raise Error.new("Translation missing for language #{locale}", path: url, patient_id: patient.id)
+    end
+
     request_body = JSON(
       phone: phone,
-      msg: I18n.t("notifications.imo.invitations.message", patient_name: patient.full_name, locale: locale),
-      contents: [{
-        key: I18n.t("notifications.imo.invitations.message_key", locale: locale),
-        value: I18n.t("notifications.imo.invitations.message", patient_name: patient.full_name, locale: locale)
-      }],
-      title: I18n.t("notifications.imo.invitations.title", locale: locale),
-      action: I18n.t("notifications.imo.invitations.action", locale: locale),
+      msg: I18n.t("notifications.imo.invitation.request", patient_name: patient.full_name, locale: locale),
+      contents: [
+        {
```

Comment:
I'm adding the name section because Daniel said it was worth keeping for notifications and I figure we should do the same for invitations.